lib/src/services/game_service.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+
+import '../models/game/game_model.dart';
+
+class GameService {
+  GameService._();
+
+  static final GameService _instance = GameService._();
+
+  static GameService get instance => _instance;
+
+  Future<GameModel> findOne(String uid) async {
+    return (await all()).firstWhere((game) => game.uid == uid);
+  }
+
+  Future<List<GameModel>> all() async {
+    String data = await rootBundle.loadString('assets/mock/data/games.json');
```

Comment:
um den Umgang mit den Mockdaten effizienter zu gestalten, könntest du einmal ganz am Anfang (im Konstruktor) die JSON laden und dann im Service einfach eine Liste halten. So könntest du in der findOne auch das direkt verwenden und müsstest nicht für einen einzigen, jedesmal ALLE auslesen. :)

Reply:
in #2 geändert worden ^^